lib/components/Textarea/Textarea.treat.ts

```diff
@@ -19,4 +19,9 @@ export const highlights = style({
   color: 'transparent !important',
   wordBreak: 'break-word',
   whiteSpace: 'pre-wrap',
+  selectors: {
+    '&:after': {
```

Comment:
Totally minor nitpick, but this is actually supported at the top level as `':after': { ... }`, so `selectors` isn't needed here.

Reply:
Ah cool, I didn't know that. I'll make that change